pkg/kubelet/config/file_test.go

```diff
@@ -91,7 +176,7 @@ func TestReadPodsFromFile(t *testing.T) {
 					Namespace: "mynamespace",
 				},
 				Spec: api.PodSpec{
-					Containers:      []api.Container{{Name: "image", Image: "test/image", SecurityContext: securitycontext.ValidSecurityContextWithContainerDefaults()}},
+					Containers:      []api.Container{{Name: "image", Image: "test/image", TerminationMessagePath: "/dev/termination-log", ImagePullPolicy: "Always", SecurityContext: securitycontext.ValidSecurityContextWithContainerDefaults()}},
```

Comment:
I revert it [here](https://github.com/wu8685/kubernetes/blob/df20a5cd4b2eaa29188efdd2d1f63ed7a7140bcd/pkg/kubelet/config/file_linux_test.go#L180)